src/odpf/proton/metadata/resource/OptimusLog.proto

```diff
@@ -0,0 +1,77 @@
+syntax = "proto3";
+
+package odpf.proton.metadata.resource;
+
+option java_package = "io.odpf.proton.metadata.resource";
+option java_outer_classname = "OptimusLog";
+option go_package = "github.com/odpf/proton/metadata/resource";
+
+import "google/protobuf/timestamp.proto";
+
+message OptimusLogKey {
+  string urn = 1;
+}
+
+message OptimusLogMessage {
```

Comment:
Optimus will start managing warehouse resources as well, like tables/views, etc. How do you think they fit here? We should have a message for Job specifically.

Reply:
renamed it to OptimusJobMetadata